amundsen_application/static/css/_buttons-default.scss

```diff
@@ -144,6 +144,22 @@
     }
   }
 
+  &.btn-link {
+    color: $link-color;
```

Comment:
I know we have some inconsistent concepts in the application regarding the styles of links vs clickable text, but I didn't think the goal here was to change the color. From the recommendation to just remove the carat, I assumed we'd still be using the usual text color with indigo on highlight.

Reply:
Yep, this was something I chat with Knowl. The current color is a grey that don't really make sense as an interactive element.

This color declaration could probably be gone, but I left it to be explicit.